docs/docs/sample-scripts/spo/copy-listitems-sharepointlist.md

```diff
@@ -0,0 +1,36 @@
+# Copy list items between SharePoint lists
+Author: [SekThang](https://twitter.com/SekThang), Inspired by [Ruud](https://lazyadmin.nl/it/copy-sharepoint-list-items-to-another-list-with-powershell-and-pnp/)
+
+Author: [SekThang](https://github.com/SekThang), Inspired by [Ruud](https://lazyadmin.nl/it/copy-sharepoint-list-items-to-another-list-with-powershell-and-pnp/)
+
+
+The cli script helps you to copy list items from one list to another list.
+I have written script logics to migrate list items from one site collection to another site collection
+
+- Prerequisites: List and metadata should be created in the destination site collection site as it's in the source site collection
+- Please update columns as per your needs in the line number 24 and 26
```

Comment:
The script only has 16 lines so we need to update the line numbers to match the lines people will see after they copy & paste the script in a new file. These should be lines 12 and 14